tooling/cli/templates/mobile/ios/project.yml

```diff
@@ -109,7 +111,9 @@ targets:
       - sdk: Security.framework
       - sdk: UIKit.framework
       {{~#each ios-frameworks}}
-      - sdk: {{this}}.framework{{/each}}
+
+      - sdk: {{this}}.framework
+      {{/each}}
       - sdk: WebKit.framework
```

Comment:
If there's no framework provided, it'll be like:
```yaml
- sdk: UIKit.framework        - sdk: WebKit.framework
```

Played around, this one should work for me:

```yaml
- sdk: UIKit.framework{{#if this.ios-frameworks}}{{~#each ios-frameworks}}
- sdk: {{this}}.framework{{/each}}{{/if}}
- sdk: WebKit.framework
```

This one also fixs for `ios-vendor-sdks` if there are multiple vendor-sdks. But since we don't support inserting `ios-vendor-sdks`, this is minor

```yaml
      - framework: {{this}}
        embed: false{{/each}}{{#if ios-vendor-frameworks}}{{~#each ios-vendor-frameworks}}
      - framework: {{this}}{{/each}}{{/if}}{{#if ios-vendor-sdks}}{{~#each ios-vendor-sdks}}
      - sdk: {{prefix-path this}}{{/each}}{{/if}}
      - sdk: CoreGraphics.framework
      - sdk: Metal.framework
      - sdk: MetalKit.framework
      - sdk: QuartzCore.framework
      - sdk: Security.framework
      - sdk: UIKit.framework{{#if this.ios-frameworks}}{{~#each ios-frameworks}}
      - sdk: {{this}}.framework{{/each}}{{/if}}
      - sdk: WebKit.framework
  ```

Reply:
nice catch, thank you! pushed your fix